crates/wasi-http/Cargo.toml

```diff
@@ -31,4 +31,4 @@ wasmtime = { workspace = true, features = ['component-model'] }
 [target.'cfg(not(any(target_arch = "riscv64", target_arch = "s390x")))'.dependencies]
 tokio-rustls = { version = "0.24.0" }
 rustls = { version = "0.21.0" }
-webpki-roots = { version = "0.23.0" }
+webpki-roots = { version = "0.25.2" }
```

Comment:
I think this (and the other `wasi-http` changes) could be separated out from this?

Reply:
Yes, If I add an exception for cargo deny I guess? I just made the updates to not have duplicated dependencies to pass Cargo deny. Is that the way to go and then a PR that makes the change and removes the exception or? I haven't used cargo deny in a work flow before so I appreciate guidance on this 🙂

Reply:
Moved into it's own PR: #6886